engine/apps/mobile_app/alert_rendering.py

```diff
@@ -10,4 +10,4 @@ def _render_for(self):
 def get_push_notification_message(alert_group):
     alert = alert_group.alerts.first()
     templated_alert = AlertMobileAppTemplater(alert).render()
-    return str_or_backup(templated_alert.title, "Alert Group")
+    return str_or_backup(f"#{alert_group.inside_organization_number} {templated_alert.title}", "Alert Group")
```

Comment:
for this @raphael-batte basically wants to mimic the format of Slack alerts ([example](https://raintank-corp.slack.com/archives/C025VMT6SPK/p1672821524988669)).

I think this may need to be modified to something like this:
```python3
return str_or_backup(f"#{alert_group.inside_organization_number} {templated_alert.title} via {alert_group.integration.name}", "Alert Group")
```
but how can I access the Integration name that fired this alert?

Reply:
I think that what you want is `{alert_group.channel.short_name}`

Reply:
But `short_name` would work too (wondering if we really need two different properties for the name)